playbooks/amc.yml

```diff
@@ -145,6 +145,7 @@
     - oauth_client_setup
     - { role: letsencrypt_role, letsencrypt_run: false }
     - { role: deploy_info, tags: ['deploy'] }
+    - check_heartbeat
```

Comment:
So it works on "fast" deployments:

```suggestion
    - { role: check_heartbeat, tags: ['deploy'] }
```

![image](https://user-images.githubusercontent.com/645156/162232878-da95417b-b75f-45f3-96a4-1c26bef088da.png)

adding the tags to individual tasks works as well, but the role-wide tag makes it more maintainable.

Reply:
I used the same set of tags and conditionals that were on the task to restart the processes: https://github.com/appsembler/configuration/blob/appsembler/juniper/master/playbooks/roles/edxapp/tasks/deploy.yml#L520-L533